src/Illuminate/Validation/Rule.php

```diff
@@ -38,7 +38,7 @@ public static function exists($table, $column = 'NULL')
      * @param  bool  $array_keys
```

Comment:
Yeah fixed that. Thanks though for the code review. It's my first time making PR so yeah 😬